mne_bids/mne_bids.py

```diff
@@ -619,6 +644,33 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                                      output_path=output_path, make_dir=False,
                                      overwrite=False, verbose=verbose)
 
+    processing = None
+    proc_history = raw.info['proc_history']
+    if proc_history != []:
+        sss = False
+        tsss = False
```

Comment:
do we need these two variables, couldn't we just assign `processing` directly?

Reply:
yep, changed :)
Not sure why I even did this in the first place...